src/main/resources/db/changelog/db.changelog-RDM-4641.xml

```diff
@@ -0,0 +1,13 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="rdm-4641" author="sateesh.mathangi@hmcts.net">
+        <addColumn tableName="case_data">
+            <column name="last_state_modified_date" type="timestamp">
+                <constraints nullable="true" />
```

Comment:
This should be changed to nullable="false", but maybe after the migration

Reply:
Yes, after migration.